user/languages/matlab.md

```diff
@@ -35,10 +36,6 @@ This guide covers build environment and configuration topics specific to
 > Currently, MATLAB builds are available only for public projects in Linux&reg; environments.
```

Comment:
Can this note be removed too?

Reply:
I think this statement is still true for Travis. It does not have the same features as our other CI offerings.